src/util/k5test.py

```diff
@@ -812,7 +814,26 @@ def _start_daemon(args, env, sentinel):
     return proc
 
 
+# Check a daemon's status prior to terminating it.  Display its return
```

Comment:
I don't know if our python style cares about declaration-before-use, but this is used on line 475 before being declared on 819.

Reply:
It's fine; since Python doesn't have forward declarations, you couldn't do mutually recursive functions if you couldn't make a forward reference.  I think it makes more sense to have it here with the other two daemon functions than before its use in _onexit().